1-exercises/A-array-find/exercise.js

```diff
@@ -5,6 +5,11 @@
 
 // write your code here
 
+function findName(names){
```

Comment:
Need to format this

Reply:
Thanks for your feedback I format the function by add a space before and after operators (=).